### DISCUSSION TITLE
有什么GitHub上的课程推荐吗？

### DISCUSSION THREAD

**Discussion by Li-Yuchen-cn** [share121/inter-knot#25] [其他]
有什么GitHub上好玩的课程推荐？最近有点无聊想找点事做，不想天天逛绳网了...

**Comment by share121** [comment]
我更推荐 B 站大学

**Comment by RS64671012** [comment]
[https://github.com/trekhleb/state-of-the-art-shitcode](https://github.com/trekhleb/state-of-the-art-shitcode)
教你如何写出屎一般的代码

**Reply by Li-Yuchen-cn** [reply]
无敌了
